.github/workflows/build_wheels_windows.yml

```diff
@@ -183,7 +183,7 @@ jobs:
             ${CONDA_RUN} python "${{ inputs.repository }}/${SMOKE_TEST_SCRIPT}"
           fi
       - name: Upload package to pytorch.org
-        if: ${{ (inputs.trigger-event == 'push' && env.CHANNEL != 'test') || (env.CHANNEL == 'test' && startsWith(github.event.ref, 'refs/tags/')) }}
+        if: ${{ (inputs.trigger-event == 'push' && env.CHANNEL != 'test' && github.event.ref != 'refs/heads/master/' && github.event.ref != 'refs/heads/main') || (env.CHANNEL == 'test' && startsWith(github.event.ref, 'refs/tags/')) }}
```

Comment:
Don't think this will work take a look at the nightly build of vision:
https://github.com/pytorch/vision/actions/runs/4588451022/jobs/8102674198
it builds from main